filters.go

```diff
@@ -194,3 +194,16 @@ func (p *Pipe) Freq() *Pipe {
 	}
 	return Echo(output.String())
 }
+
+// Column reads from the pipe, and returns a new pipe containing only the Nth
+// column of each line in the input (where columns are delimited by whitespace).
+// If there is an error reading the pipe, the pipe's error status is also set.
+func (p *Pipe) Column(col int) *Pipe {
+	return p.EachLine(func(line string, out *strings.Builder) {
+		columns := strings.Fields(line)
+		if col <= len(columns) {
+			out.WriteString(columns[col-1])
+			out.WriteByte('\n')
```

Comment:
This makes me think about Windows based systems, where it would maybe more obvious to emit `\r\n`. Do you have some plans to support these as well?

Reply:
That's worth thinking about, but not being a Windows user myself, I'm not sure what the right thing is to do. [Some discussion about `gofmt` line endings here](https://github.com/golang/go/issues/16355).

Reply:
If I'm interpreting https://github.com/golang/go/issues/28822 correctly, then `Println` does not emit different line endings for Windows. So I don't think we should do that here either.

(It's conceivable that somebody could write a `script` filter which turns LF endings into CRLF, in which case Windows users can just filter everything through that if they need the CRs.)